content/SCALE/SCALETutorials/SystemSettings/Services/SMBServiceSCALE.md

```diff
@@ -13,41 +13,30 @@ tags:
 The **Services > SMB** screen displays after going to the **Shares** screen, finding the **Windows (SMB) Shares** section, and clicking <span class="material-icons">more_vert</span> + **Config Service**.
 Alternately, you can go to **System Settings > Services** and click the <span class="material-icons">edit</span> edit icon for the SMB service.
 
-## SMB Services Screen
-The **SMB Services** screen displays setting options to configure TrueNAS SMB settings to fit your use case.
-The **Basic Options** settings continue to display after selecting the **Advanced Options** screen.
+## Configuring SMB Service
+The **SMB Services** screen displays setting options to configure TrueNAS SMB settings to fit your use case. 
+In most cases you can set the required fields and accept the rest of the setting defaults, but if you have specific needs for your uses case click **Advanced Options** to display more settings.
```

Comment:
Suggestion for line 18:
"In most cases you can set the required fields and accept the rest of the setting defaults.  If you have specific needs for your uses case, click **Advanced Options**.  This  will display more settings."